### ISSUE TITLE
Bad Method Signature

### ISSUE THREAD

**Issue by aarondfrancis** [lob/lob-php#88]
```
Declaration of Lob\Resource\USVerifications::create(array $data) must be compatible with Lob\Resource::create(array $data, ?array $headers = NULL)
```

The US Verifications class extends the Lob\Resource class, but the create signatures don't match

**Comment by aarondfrancis** [comment]
@mpiercy827 fixed or just closed?

**Comment by mpiercy827** [comment]
@aarondfrancis This was fixed in [#89](https://github.com/lob/lob-php/pull/89), and is included in v2.2.0.

**Comment by aarondfrancis** [comment]
Awesome, thanks!
